src/ui/server/controllers/insurance/policy/map-submitted-data/broker/index.ts

```diff
@@ -31,6 +31,16 @@ const mapSubmittedData = (formBody: RequestBody): object => {
     delete populatedData[USING_BROKER];
   }
 
+  /**
+   * If IS_BASED_IN_UK is an empty string,
+   * nullify IS_BASED_IN_UK related fields
+   */
+  if (isEmptyString(formBody[IS_BASED_IN_UK])) {
```

Comment:
Might not need this if we default it to `false`.

Reply:
We can't have a default as per comment above